lib/tasks/worker.rake

```diff
@@ -3,11 +3,27 @@
 namespace(:worker) do
   desc("Run the worker")
   task(run: :environment) do
-    # See https://googleapis.dev/ruby/google-cloud-pubsub/latest/index.html
+    puts "worker begins"
 
-    puts("Worker starting...")
+    pubsub = Pubsub.new
+
+    subscription = pubsub.subscription('default')
```

Comment:
This is one of the queue names. What if we had multiple?

Reply:
Added the queue as an argument to handle multiple queues